goto/commands/open.py

```diff
@@ -28,4 +28,4 @@ def open(magic, args):
     except subprocess.CalledProcessError as e:
         return None, GotoError("open_failed", message=e.message)
 
-    return None, None
+    return "%s" % url, None
```

Comment:
1. the % syntax here is unecessary, could just use `url`
2. I would want this output to be shown only when `-v` or `--verbose` is added to the command
In order to keep consistent behaviour.  
`verbose = '-v' in args or '--verbose' in args`
`output = output if verbose else None`